### PR TITLE
Update all modules, fix template points

### PR DESCRIPTION
**Pull request by apeltzer**
## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/scrnaseq/tree/master/.github/CONTRIBUTING.md)- [ ] If necessary, also make a PR on the nf-core/scrnaseq _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [x] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: nextflow.config [hunk 1]
```diff
@@ -111,15 +110,13 @@ try {
 profiles {
     debug { process.beforeScript = 'echo $HOSTNAME' }
     conda {
-        conda.enabled          = true
```
Comment (FriederikeHanssen) [inline nextflow.config:114]: why are you removing this?

Comment (klkeys) [reply nextflow.config:114]: possibly deprecated? it looks like the `conda` configuration scope no longer includes an option `enabled`

https://www.nextflow.io/docs/latest/config.html#scope-conda

Comment (FriederikeHanssen) [reply nextflow.config:114]: I thought this was just fix in December with adding this https://github.com/nf-core/tools/blob/dev/nf_core/pipeline-template/nextflow.config (after the `enabled_conda` param wasn't necessary anymore) 🤔 . If that is the case, should be added to the template too, I guess

Comment (apeltzer) [reply nextflow.config:114]: Yeah sorry, I think I confused it with that enabled_conda param that was deprecated...

**Review by grst — APPROVED** [review]
I re-added the `conda.enabled` lines.

Rest looks good!

**Pull request merged**